src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/PrisonerProfileController.kt

```diff
@@ -53,4 +56,40 @@ class PrisonerProfileController(
   fun getPrisonerProfile(@PathVariable prisonId: String, @PathVariable prisonerId: String): PrisonerProfileDto? {
     return prisonerProfileService.getPrisonerProfile(prisonId, prisonerId)
   }
+
+  @PreAuthorize("hasRole('ROLE_ORCHESTRATION_SERVICE__VISIT_BOOKER_REGISTRY')")
```

Comment:
Is this the role the Public UI will need to use for this endpoint? If so, I _think_ it should be `ROLE_VISIT_ORCHESTRATION__PUBLIC_VISIT_BOOKING_UI` to match the proposed convention 🤔. Perhaps worth a chat...